app/src/hooks/useGetProjects.ts

```diff
@@ -0,0 +1,16 @@
+import useAxios from 'axios-hooks';
+import React from 'react';
+
+function useGetProjects() {
+  const [{ data, error, loading }, refetch] = useAxios(
+    'http://localhost:4000/api/v1/project/mock/data',
```

Comment:
endpoint hardcoded here, is it intentional? @roy-sagar11 
Can we put the endpoint in some config, maybe?

Reply:
Either config, or use a relative address `/api/v1/...` for local development as well (you'd need to use the react proxy I think)

Reply:
@roy-sagar11 @matbusby-fw We can use axios `interceptors` for centralise  configuration

Reply:
ohh .. that was my mistake. i was working on a different branch and stashed the changed but forgot to add this new files to git. so it didn;t get shashed and came with the fix branch. i'll open a pr with this changes later